index.njk

```diff
@@ -2,19 +2,54 @@
 layout: 'layouts/base.njk'
 ---
 
-<h1>
-  <span class="eyebrow">Design Sprint Guide</span>
-  Design, prototype, and test ideas quickly.
-</h1>
+<main class="tbds-app-frame" "tbds-block-stack" "tbds-block-stack--bordered">
+  
+  <header class="tbds-app-frame__header  tbds-margin-block-6">
+    <section class="tbds-inline-stack tbds-inline-stack--gap-6 tbds-inline-size-80%">
```

Comment:
I'm not sure that `tbds-inline-stack--gap-6` is having any effect here, perhaps remove it?

Also I wonder if we could remove `tbds-inline-size-80%` so that the nav is the full width of the page?